content/en/blog/2024/otel-collector-survey/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Insights From The OpenTelemetry Collector Survey
```

Comment:
```suggestion
title: Insights from the OpenTelemetry Collector Survey
```
The OTel docs style guide looks like it suggests lowercasing "from" and "the" for title case.